http.go

```diff
@@ -0,0 +1,112 @@
+// Copyright 2016 Comcast Cable Communications Management, LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// End Copyright
+
+package money
+
+import (
+	"bytes"
+	"net/http"
+)
+
+//Header keys
+const (
+	MoneyHeader      = "X-Money-Trace"
+	MoneySpansHeader = "X-Money-Spans"
+
+	//money-trace context keys
+	tIDKey = "trace-id"
```

Comment:
Context keys should be internal, distinct types.  This avoids namespace collision.  For example:

```
type contextKey string

const (
    tIDKey contextKey = "trace-id"

    // etc.
)
```

Reply:
These are not golang context keys. They're used for decoding values from headers.